### ISSUE TITLE
LUA: Fix sanitizer problems for the witcher/lua

### ISSUE THREAD

**Issue by Nostritius** [xoreos/xoreos#385]
The witcher has some sanitizer problems related to the lua part, as written in #384.

**Comment by yoshimo** [comment]
the  pr is closed , i guess the issue can be closed

**Comment by DrMcCoy** [comment]
@yoshimo Not really, no. The PR is where we *discovered* the sanitizer issues. I had asked @Nostritius to open the issue in a comment on the PR: https://github.com/xoreos/xoreos/pull/384#issuecomment-439930976
